src/extension/src/CoreTSModule.ts

```diff
@@ -0,0 +1,34 @@
+import { WizardServant, IPayloadResponse } from "./wizardServant";
+import { ExtensionCommand } from "./constants";
+import { CoreTemplateStudio } from "./coreTemplateStudio";
+
+export class CoreTSModule extends WizardServant {
+    clientCommandMap: Map<
+        ExtensionCommand,
+        (message: any) => Promise<IPayloadResponse>
+    >;
+
+    constructor() {
+        super();
+        this.clientCommandMap = this.defineCommandMap();
+        console.log("creating CoreTSmodule");
```

Comment:
Do we need this console log or is it for debugging?

Reply:
updated PR